src/resources/views/post/show.blade.php

```diff
@@ -4,42 +4,63 @@
 
     @include('layouts.navbar')
 
-    <div class="container mt-4">
-        <div class="card" style="background-color: #FAF9F6; border: 2px solid #FFA903;">
-            <div class="card-header d-flex justify-content-between align-items-center" style="background-color: #FFA903">
-                <div class="text-muted small">
+    <div class="container mt-5">
+        <div class="row justify-content-center">
+        <div class="col-md-7">
+        <div class="card " style="background-color: #F6f6f2; border: 1px solid #388087;">
+            <div class="card-header d-flex justify-content-between align-items-center" style="background-color: #6fb3b8; border-bottom: 1px solid #388087;">
+                <div class="text-white small">
                     Created at: {{ $post->created_at->setTimezone('Asia/Manila')->format('j M Y \a\t g:ia') }}
                 </div>
-                <div class="text-muted small">
+                <div class="text-white small">
                     Updated at: {{ $post->updated_at->setTimezone('Asia/Manila')->format('j M Y \a\t g:ia') }}
                 </div>
             </div>
             <div class="card-body">
-                <h3 class="card-title">{{ $post->user->name }}</h3>
-                <p class="card-text" style="font-size: 20px;">{{ $post->content }}</p>
+                <div class="d-flex align-items-center mb-3">
+                    <img src="https://cdn-icons-png.flaticon.com/512/456/456283.png" alt="{{ $post->user->name }} Profile Picture" class="rounded-circle" width="35" height="35">
+                    <div class="ms-3">
+                        @if (auth()->check() && $post->user->id === auth()->user()->id)
+                            <a class="card-title text-decoration-none h4" href="{{ route('profile.show') }}" style="color: #388087;">
+                                {{ $post->user->full_name }}
+                            </a>
+                        @else
+                            <a class="card-title text-decoration-none h4" href="{{ route('profile.index', $post->user) }}" style="color: #388087;">
+                                {{ $post->user->full_name }}
+                            </a>
+                        @endif
```

Comment:
It seems like the same output but a different redirection link.

You can use ternary operator inside href like this

```blade
  @php
   $ownPost =  auth()->check() && $post->user->id === auth()->user()->id
  @endphp
  <a class="card-title text-decoration-none h4" href="{{ $ownPost ? route('profile.show') : route('profile.index') }}" style="color: #388087;">
  {{ $post->user->full_name }}
  </a>
```

Reply:
I have made changes on my PR, sir. Thank you!